basis/models.py

```diff
@@ -35,9 +35,12 @@ class PersistentModel(models.Model):
     class Meta:
         abstract = True
 
-    def delete(self, *args, **kwargs):
-        self.deleted = True
-        self.save(*args, **kwargs)
+    def delete(self, using=None, force=False):
```

Comment:
On the other hand, it might be possible to mock `Model.delete()` and check what it is called with.